src/SprykerEco/Zed/UnzerGui/Communication/Table/MerchantUnzerCredentialsTable.php

```diff
@@ -104,7 +104,7 @@ protected function setHeader(TableConfiguration $tableConfiguration): TableConfi
     {
         $tableConfiguration->setHeader([
             SpyUnzerCredentialsTableMap::COL_ID_UNZER_CREDENTIALS => 'ID',
-            SpyUnzerCredentialsTableMap::COL_CONFIG_NAME => 'Config name',
+            SpyUnzerCredentialsTableMap::COL_CONFIG_NAME => 'Name',
```

Comment:
Please, check that you have it in the module's translation file.

Reply:
fixed